osu.Game/Skinning/GameplaySkinComponentLookup.cs

```diff
@@ -28,8 +27,5 @@ public GameplaySkinComponentLookup(T component)
 
         protected virtual string RulesetPrefix => string.Empty;
         protected virtual string ComponentName => Component.ToString();
-
-        public string LookupName =>
-            string.Join('/', new[] { "Gameplay", RulesetPrefix, ComponentName }.Where(s => !string.IsNullOrEmpty(s)));
```

Comment:
I have removed this because the one and only usage of this (in `LegacyReverseArrow`) has just been removed. Also this whole path of lookup is becoming ancient with the concept of using a `SkinTransformer` for ruleset components (e.g. `ArgonSkinTransformer`).

Reply:
Very glad to see this go.